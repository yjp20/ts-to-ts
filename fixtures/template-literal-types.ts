/** @model */
type Greeting = `hello ${string}`;
/** @model */
type EventName = `${string}:click` | `${string}:hover`;
/** @model */
type Color = `#${string}`;
