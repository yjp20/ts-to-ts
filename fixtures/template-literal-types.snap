
<<<<<<<<<<< typescript <<<<<<<<<<<<
/** @model */
type Greeting = `hello ${string}`;
/** @model */
type EventName = `${string}:click` | `${string}:hover`;
/** @model */
type Color = `#${string}`;
===================================
// template-literal-types.ts
alias Greeting = "hello ${string}";

alias EventName = "${string}:click" | "${string}:hover";

alias Color = "#${string}";
>>>>>>>>>>> typespec >>>>>>>>>>>>>>